### PR TITLE
Updates to field visit information in examples

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

File: inst/extdata/correctionsataglance-example.json [hunk 1]
```diff
@@ -103,267 +103,187 @@
...
+    "altitudeDatumCode": "NGVD29    ",
+    "siteNumber": "06933500       ",
+    "stationName": "Gasconade River at Jerome, MO",
+    "coordinateDatumCode": "NAD83     ",
+    "state": "MISSOURI",
+    "startDate": "2014-12-24T00:00:00-06:00",
+    "stationId": "06933500"
+  },
+  "fieldVisits": [
     {
```
Comment (dpattermann-usgs) [inline inst/extdata/correctionsataglance-example.json:134]: So field visits are taking the place of readings?

File: inst/extdata/correctionsataglance-example.json [hunk 2]
```diff
@@ -103,267 +103,187 @@
...
-      "measurementNumber": "959",
-      "qualityRating": "FAIR",
-      "historic": false,
-      "meanGageHeight": 14.45,
-      "meanGageHeightUnits": "ft",
-      "shiftNumber": 0
+      "isValid": true,
+      "lastModified": "2015-11-29T03:13:14.845-06:00",
+      "party": "tk,cp",
+      "remarks": "loop rating study qm.",
```
Comment (dpattermann-usgs) [inline inst/extdata/correctionsataglance-example.json:142]: Do these take the place of comments?

File: inst/extdata/correctionsataglance-example2.json [hunk 3]
```diff
@@ -155,27 +99,424 @@
...
+      "isValid": true,
+      "lastModified": "2015-11-29T01:23:24.116-06:00",
+      "party": "MDM",
+      "remarks": "Preemptive strike, proactive nitratax battery swap before batteries die over the long joyous holiday weekend break leading into 2015. Oh! what mysteries await us all in the new year; how will our equipment fare situated in the mighty Kansas River of DeSoto. Father Time tics on, and the existential conundrums shall continue to elude us until the moment of truth arrives, as so here is hoping that our little EXO and Nitratax cherish every moment and make the most out of it, taking the time to enjoy the little things like the majestic eagles\u0027 uncontrollable freedom of flight over the river, or the sound of swallow droppings plopping in the river all around the sonde installation, because we never know what debris may lay around the corner, or the shifting sand banks that may one day overcome the installation site and bring about change; but then, change is not always necessarily a bad thing. One last thing of note, the river was full of moving ice, still clear, ice free, moving water around the sonde and pier that it is situated behind.",
```
Comment (dpattermann-usgs) [inline inst/extdata/correctionsataglance-example2.json:144]: This is a good test for how long comments look.

File: inst/extdata/correctionsataglance-example.json (hunk 1 above)
Comment (thongsav-usgs) [inline inst/extdata/correctionsataglance-example.json:134]: Yes, this report is interested in top level field visits, not readings or measurements.

File: inst/extdata/correctionsataglance-example.json (hunk 2 above)
Comment (thongsav-usgs) [inline inst/extdata/correctionsataglance-example.json:142]: No, we are no longer concerned with readings in this data, they are Field Visit level remarks.

File: inst/extdata/correctionsataglance-example2.json (hunk 3 above)
Comment (thongsav-usgs) [inline inst/extdata/correctionsataglance-example2.json:144]: We also may want to convert newlines "\n" to "BR" tags in the final HTML rendering. That may have fixed the rendering issues in the SRS report.

**Pull request merged**